paper/paper.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'secuTrialR: Seamless interaction with clinical trial databases in R'
+tags:
+  - R software
+  - clinical trials
+  - data management
+  - descriptive statistics
+  - secuTrial
+authors:
+ - name: Patrick R. Wright
+   orcid: 0000-0002-1153-0846 
+   affiliation: "1, 3"
+ - name: Alan G. Haynes
+   orcid: 0000-0003-1374-081X
+   affiliation: "2, 3"
+ - name: Milica Markovic
+   orcid: 0000-0002-6973-6679
+   affiliation: "1, 3"
+affiliations:
+ - name: University Hospital Basel, Clinical Trial Unit, Basel, Switzerland
+   index: 1
+ - name: University of Bern, Clinical Trial Unit, Bern, Switzerland
```

Comment:
It needs to be "CTU Bern, University of Bern", otherwise our library people may not pick it up. I wouldnt bother about adding the cities to the affiliations either - in the case of the universities its obvious

Reply:
Funny enough, the ETH Zürich is opening branches in Basel :D 

I'll adjust yours but leave ours, since that's apparently the structure we are supposed to use.